src/hardware/sblaster.cpp

```diff
@@ -309,7 +311,8 @@ static int e2_incr_table[4][9] = {
 };
 
 static int frames_added_this_tick = 0;
-RWQueue<std::unique_ptr<AudioVector>> soundblaster_mixer_queue{128};
+
+RWQueue<AudioFrame> soundblaster_mixer_queue{128};
```

Comment:
The size needs to be increased.  Previously, each element was about 1ms of audio (so 128ms total buffer).  Now this is just 128 frames (less than 3ms at a typical sample rate).

Ideally this can re-use the generic `MIXER_PullFromQueueCallback` which re-sizes it dynamically based on frames per block and whether we're currently in fast-forward mode.

Reply:
> Ideally this can re-use the generic `MIXER_PullFromQueueCallback` which re-sizes it dynamically based on frames per block

Moved the SB to use MIXER_PullFromQueueCallback, so this global `soundblaster_mixer_queue` is now gone like you suggested.